backend/services/accounts.js

```diff
@@ -1,26 +1,31 @@
-import mockedAccounts from '../mockData/accounts.js';
+import Account from '../models/accounts.js';
 
 export default class AccountsService {
-  static getAccounts() {
-    return mockedAccounts;
+  static getAccounts(userId) {
+    const accounts = Account.find({ userId });
+    return accounts;
   }
 
-  static getAccount(id) {
-    const account = mockedAccounts.find((acc) => acc.id === id);
+  static getAccount(accountId, userId) {
+    const account = Account.findOne({ _id: accountId, userId });
```

Comment:
method Account.findOne({ _id: accountId, userId }); returns promise. Did you forget 'await'?